### PR TITLE
Render sprites and images

### LINKED ISSUE #3: DF Artifact + Spaceship images are missing
CleanShot 2023-07-03 at 13 29 58@2x

The prev url: https://d2wspbczt15cqu.cloudfront.net/v0.6.0-artifacts/64-mothership.png is no longer working.

### PR DESCRIPTION
**Pull request by 0xhank**
suboptimal solution using a ratchet npm package to get sprite information

i will optimize the spritesheet soon but for now this is passable at least

Closes #3

### CODE REVIEW

**Comment by 0xhank** [comment]
The sprites are still lo-res but i think thats okay for an icon in the corner. Now they are perfectly responsive.

**Review by cha0sg0d — APPROVED** [review]
This is so clutch! One question - why do we need to remove the `public`?

File: client/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@dfdao/dynasty": "1.1.0",
     "@lit-labs/react": "^1.0.0",
     "@radix-ui/react-toast": "^0.1.1",
+    "@types/react-responsive-spritesheet": "^1.0.0",
```
Comment (cha0sg0d) [inline client/package.json:26]: @0xhank do we need this still?

**Comment by 0xhank** [comment]
It's inferred by the browser so there can be compatibility issues if left in

**Pull request merged**